traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -295,7 +295,7 @@ sub update {
 		my @response;
 		push(
 			@response, {
-				"active"                   => \$rs->active,
+				"active"                   => $rs->active,
```

Comment:
This is on the update method.  When the result set was being returned after the update, the active flag (and the others with the `\` removed) were not being rendered correctly.  Removing the `\` caused the field to be returned correctly.